test/packages/with-kind/kubernetes/_dev/deploy/k8s/daemonset.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
```

Comment:
where is this daemonset.yaml coming from?

Reply:
We need these resources to check the `state_*` datastreams, but since I only included one for the pod, maybe I should delete all the others? I was trying to follow the same logic as the one that was here before.

Reply:
I am torn between deleting some of the resources because they are useless, or keeping them so they stay the same as in the integrations repo

Reply:
let me see if I understand. 

you are adding a state_pod datastream here since `pod` didn't even need the kube-state-metrics resource that was already defined in those manifests resources (before the kustomization was introduced). those resources in deploy/k8s are in theory used to setup the environment for testing but since we don't check for output against an expected result, they are kind of useless. we only check that the setup didn't have any issues.

I would say, keep them in sync with what we are doing in the integrations repo. it's definitively another PR

Reply:
Yes, they were useless before because the only data stream defined had the `.empty` file. They are no longer useless.